source/TimePicker.js

```diff
@@ -479,10 +479,24 @@
 				default:
 					break;
 				}
-	
 			}
 	
 			this.inherited(arguments);
+
+			// in case of timePicker, timeWrapper enclose hour and minute pickers
+			// so, we re-organize this.pickers
+			var pickers = this.pickers;
```

Comment:
Hi @aarontam , Thank you for your review.
Unfortunately, I didn't consider much today due to pre.11.1 release.
But I made a simple change.
Could you check this is a same way which you commented?
Thank you.